### PR TITLE
Product categories

### LINKED ISSUE #1: Create product categories
## Value statement - a requirement, not a solution
As a **user** 
I want **an overview of product categories** 
so that **I can find products more easliy through an enclosed category** 

## Description (Text, Scribble, Wireframe, Design)
Screenshot 2020-08-28 at 12 02 15

## Acceptance criteria
- [x] All necessary categories are listed on the site
- [x] All categories can be clicked
- [x] All categories lead to a new site

## Tasks
- [x] Create feature branch (feat/product-categories)
- [x] Create Category component in Storybook
- [x] Write component tests
- [x] Add components to App

## Complexity
small

### LINKED ISSUE #3: Link the categories with products
## Value statement - a requirement, not a solution
As a **user** 
I want **to click on a category** 
so that **I see all the products connected to the category** 

## Description (Text, Scribble, Wireframe, Design)
Screenshot 2020-08-28 at 12 09 02

## Acceptance criteria
- [x] Clicking on a category leads the user to the associated products
- [x] Each category is added to the root URL e.g. `/burgers`
- [x] Headline states the name of the category
- [x] Arrow let's you return to the category page

## Tasks
- [x] Create router for every category
- [x] Update existing component in Storybook
- [x] Test the updated component
- [x] Add component to the app

## Complexity
medium

### PR DESCRIPTION
**Pull request by denniswitzel**
This pull request is linked to user story #1  and #3 

You can ignore the snapshot files and the SVGs

### CODE REVIEW

File: src/components/ProductOverview/ProductOverview.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+}
+
+ProductOverview.defaultProps = {
+    productIcon: 'https://raw.githubusercontent.com/denniswitzel/capstone-project/f892d3d147b3c573a796dcb1a34298351806d8b5/src/images/icons/burger.svg',
+    productName: 'BURGER',
+}
+
+export default function ProductOverview({productIcon, productName}) {
+    return (    
+        <NavLinkStyled to={`/${productName.toLowerCase()}`}>
```
Comment (constantingoeldel) [inline src/components/ProductOverview/ProductOverview.js]: Is the template literal necessary? How about: 
```suggestion
        <NavLinkStyled to={productName.toLowerCase()}>
```

File: src/pages/Kebab/Kebab.js [hunk 2]
```diff
@@ -1,13 +1,14 @@
 import React from 'react'
 import kebabs from '../../mocks/kebab.json'
-import Product from '../Products/Product'
+import Product from '../../components/Products/Product'
+import Headline from '../../components/Headline/Headline'
 import PropTypes from 'prop-types'
-import Headline from '../Headline/Headline'
+
```
Comment (constantingoeldel) [inline src/pages/Kebab/Kebab.js]: For every product category, you use a very similar function to build its page, e.g. `Kebab.js` and `Burger.js` are nearly identical. You might want to use a utility function to streamline this and reduce the work needed to add new pages in the future.

File: src/App.js [hunk 3]
```diff
@@ -1,27 +1,33 @@
...
+                <Route exact path="/"> 
+                <LogoStyled />
+                {productCategories.map((category, index) => 
+                <ProductOverview key={index} productIcon={category.icon} productName={category.name}/>)}
+                </Route>
+                <Route path="/burger" component={Burger}/>
+                <Route path="/kebab" component={Kebab}/>
+                <Route path="/minced" component={Minced}/>
+                <Route path="/nuggets" component={Nuggets}/>
+                <Route path="/sausages" component={Sausages}/>
```
Comment (constantingoeldel) [inline src/App.js]: You currently don't have a route for cold-cuts, when I click on the icon, I get referred to a blank page

**Review by constantingoeldel — APPROVED** [review]
You could reduce the amount of files by a bit but otherwise everything looks good to me :) Love the minimalist design and could see myself using the app regularly as I often have to decide between meat-like products and don't know about their quality beforehand

File: src/components/ProductOverview/ProductOverview.js (hunk 1 above)
Comment (denniswitzel) [reply src/components/ProductOverview/ProductOverview.js]: Thanks for that! It actually is not necessary to do it the way I did it.

File: src/App.js (hunk 3 above)
Comment (denniswitzel) [reply src/App.js]: I currently have no data for it. I guess I should put a "Coming soon" in there as long as there is nothing to show.

**Pull request merged**